lib/mysql.js

```diff
@@ -459,6 +460,19 @@ MySQL.prototype.ping = function(cb) {
   this.execute('SELECT 1 AS result', cb);
 };
 
+MySQL.prototype.buildExpression = function(columnName, operator, columnValue,
+    propertyValue) {
+  if (operator === 'regexp') {
+    // strip regex flags because mysql doesn't allow them
+    var regex = utils.getRegExpExpression(columnValue);
```

Comment:
What does `utils.getRegExpExpression(columnValue);` do? `propertyValue` should already have the value of RegExp.